src/sparseml/tensorflow_v1/models/registry.py

```diff
@@ -151,21 +153,27 @@ def create_zoo_model(
             pretrained if isinstance(pretrained, str) else attributes.default_desc
         )
 
-        return Zoo.load_model(
-            attributes.domain,
-            attributes.sub_domain,
-            attributes.architecture,
-            attributes.sub_architecture,
-            TENSORFLOW_V1_FRAMEWORK,
-            attributes.repo_source,
-            attributes.default_dataset
+        model_dict = {
+            "domain": attributes.domain,
+            "sub_domain": attributes.sub_domain,
+            "architecture": attributes.architecture,
+            "sub_architecture": attributes.sub_architecture,
+            "framework": TENSORFLOW_V1_FRAMEWORK,
+            "repo": attributes.repo_source,
+            "dataset": attributes.default_dataset
             if pretrained_dataset is None
             else pretrained_dataset,
-            None,
-            sparse_name,
-            sparse_category,
-            sparse_target,
-        )
+            "sparse_tag": f"{sparse_name}-{sparse_category}",
+        }
+        # clear cache
+        CACHE_DIR = os.path.expanduser(os.path.join("~", ".cache", "sparsezoo"))
```

Comment:
same cache question (only need to respond once, just noting for review)

Reply:
proposal mentioned in other comment.